task/import.go

```diff
@@ -75,10 +76,13 @@ func TaskImport(tctx *TaskContext) (*data.TaskOutput, error) {
 		return nil, err
 	}
 	defer importedFile.Close()
-	// RecordStream on output file for HLS playback
-	playbackRecordingId, err := Prepare(tctx, metadata.AssetSpec, importedFile)
-	if err != nil {
-		glog.Errorf("error preparing imported file assetId=%s err=%q", tctx.OutputAsset.ID, err)
+	playbackRecordingId := ""
+	if strings.Contains(params.URL, "/recordings/") {
```

Comment:
WDYT of adding a livepeercdn prefix check as well?
```suggestion
	if strings.HasPrefix(params.URL, "https://livepeercdn.") && strings.Contains(params.URL, "/recordings/") {
```